sources/packages/web/src/components/common/notes/CreateNoteModal.vue

```diff
@@ -1,44 +1,68 @@
 <template>
-  <modal-dialog-base :showDialog="showDialog" @dialogClosed="dialogClosed">
-    <template v-slot:content>
-      <v-container class="min-width-modal">
-        <formio
-          formName="createnote"
-          @loaded="formLoaded"
-          @submitted="submitNote"
-        ></formio>
-      </v-container>
-    </template>
-    <template v-slot:footer>
-      <check-permission-role :role="allowedRole">
-        <template #="{ notAllowed }">
-          <footer-buttons
-            :disablePrimaryButton="notAllowed"
-            primaryLabel="Add note"
-            @primaryClick="addNewNote"
-            @secondaryClick="dialogClosed"
-          />
-        </template>
-      </check-permission-role>
-    </template>
-  </modal-dialog-base>
+  <v-form ref="addNewNoteForm">
+    <modal-dialog-base
+      title="Create new note"
+      :showDialog="showDialog"
+      max-width="730"
+    >
+      <template #content>
+        <error-summary :errors="addNewNoteForm.errors" />
+        <div class="pb-2">
+          <span class="label-value"
+            >Add a note with relavent decisions or actions taken on this
+            account.</span
+          >
+        </div>
+        <v-autocomplete
+          label="Note type"
+          density="compact"
+          :items="noteTypeItems"
+          v-model="formModel.noteType"
+          variant="outlined"
+          placeholder="Select a Note type"
+          :rules="[(v) => !!v || 'Note type is required.']" />
+        <v-textarea
+          hide-details
+          label="Note body"
+          placeholder="Long text..."
+          v-model="formModel.description"
+          variant="outlined"
+          :rules="[(v) => !!v || 'Note body is required']"
```

Comment:
Note body must be validated for 500 characters maximum. 
![image](https://user-images.githubusercontent.com/54600590/193674950-e256a216-040e-46e7-a191-0e136bdd339f.png)